### ISSUE TITLE
Baha bow power is pointless. and weapon powers causing class powers recharge lag.

### ISSUE THREAD

**Issue by i982** [torilmud/issues#778]
for ranger archery (as murky as it still is) there is no room in your rotation for this to trigger a powers recharge on your ranger class powers.

reason 1: if you use the arrow of slaying as a opener. if mob is caster you will be stuck in powers lag past first spell going off.

reason 2: fights arent long enough to warrant the cooldown on it vs the 10d10. unless you use your spell-up before every single fight (good luck with that Lilithelle will kill you in zone) you will never seen most of your abilities anyways. much less have time for a new extra one.

reason 3: arrow of slaying vs class powers. only one you can sacrifice (lowest damage) is knockdown shot (which needs to miss) otherwise is better than arrow of slaying (plus its your knockdown)

reason 4: with new changes to ranger powers recharges it is not even a good idea to try and fit arrrow of slaying in. it just wont. you are cause the rotation to be over complicated. Shevrash said the classes are supposed to have a flow to their rotation.. archery doesnt. and adding in more powers / class recharge lag is not conducive.

i can post logs. but its pretty easy to see if you wanna test it yourself.

**Comment by Dartan82** [comment]
been saying this for daa axe and stun bulwark

**Comment by i982** [comment]
my suggestion is make it a proc like a normal weapon... or change it to something different. the bow hardly procs to begin with. 

i tell you what a nice idea would be.. why not a proc that has a % on hit to give you a 2nd/3rd/4th/5th circle charge back on your spells at random? (just one spell at random obviously.)

id even take a vamping bow / proc heal.. since every other melee class has the option cept archery. i mean archery doesnt get enough attacks in the first place to make it as good as dual wielding a scepter/graven or 2 gravens...

only advantage bow offers is getting around shielded mobs which with items procs/enchanter/SM quest dont matter anyways. could even be a proc that breaks shields... that would be super useful.

**Comment by jokkmeister** [comment]
i982 Vamping/heal bow is Dragonhunter bow. A bow that is harder to get than Baha bow. That being said of course Baha bow's power should work but it shouldn't be better than 3rd or 4th of the bows in game. Icicles should be best followed by Dragonhunter, then I can see Baha bow. There are already more Baha bows in game then there is Dragonhunter and Icicles combined, these are two bows in need of upgrades.

**Comment by i982** [comment]
uhh wasnt really suggesting making the bow better. just changing the powers to a proc or removing the power use from the class powers cooldown table.

**Comment by Dartan82** [comment]
Ikhar let's not confuse whark bow and dhunter bow being garbage and a bar of what top bows should be.  Dhunter has no hit/dam, no damage proc, and hurts you for 2 hp for nothing so it's pure garbage.  Whark bow is still the best bow but when you compare it to all the awesome 2h weapons (deathknell, baha blades) and 1h (swiftwind, graven, valhalla) it is terrible.  Whark bow is something that would have been awesome in the 90's.  

All Tukin is asking is Bahamut bow to not suck.  

< 724h/739H 270v/270V |tr +|ks +|sp +|cn +|ha +| |3|3|3|3|2| T: Rolanis TC: few scratches E: bearded EC: pretty hurt EP: knl P: std > 
A momentary brilliant glow originating from your circlet begins
a rapid change in your appearance. Countless serpents begin to uncoil,
slithering and sliding their way down and around your arms, neck,
and torso, granting you a reptilian quickstrike ability! Your eyes gleam
a vivid yellow as the brief enchantment begins to fade. Quickly as
they descended, the serpents twist and twine their way back to the magic
within their circlet.
Your massive shot wounds a bearded commoner harshly.
As you fire, energy from a dragonhunter longbow washes over you.
A bearded commoner dodges your futile attack.
As you fire, energy from a dragonhunter longbow washes over you.
Your massive shot wounds a bearded commoner harshly.
Your massive shot brutally wounds a bearded commoner.
A bearded commoner staggers from your massive shot!
As you fire, energy from a dragonhunter longbow washes over you.
You critically wound a bearded commoner with a massive shot!
As you fire, energy from a dragonhunter longbow washes over you.
You enshroud a bearded commoner in a mist of blood with your massive shot!
< 717h/739H 270v/270V |tr +|ks +|sp +|cn +|ha +| |3|3|3|3|2| T: Rolanis TC: few scratches E: bearded EC: awful EP: knl P: std > 
A bearded commoner dodges your futile attack.
A bearded commoner is nearly slain by the force of your massive shot!
A bearded commoner misses you with his hit.
< 717h/739H 270v/270V |tr +|ks +|sp +|cn +|ha +| |3|3|3|3|2| T: Rolanis TC: few scratches E: bearded EC: awful EP: knl P: std > 
As you fire, energy from a dragonhunter longbow washes over you.
You successfully pierce a bearded commoner, his lifeless body falls to the ground.
a bearded commoner is dead! R.I.P.
You receive 0 XP at Time: Monday July 27, 2020 8:56:53 pm (0.00%) experience.
Your blood freezes as you hear the rattling death cry of a bearded commoner.
< 715h/739H 270v/270V |tr +|ks +|sp +|cn +|ha +| |3|3|3|3|2| P: std >

**Comment by jokkmeister** [comment]
Yeah all with you there. My point was mainly that Dhunter bow has heal proc and was meant to be vampiric so Baha bow should not get that effect. And also while I fully agree that Baha bow should not suck and of course should have a working power, I want them to fix dhunter and icicles bows if they start doing fixes to bows. I should rather have said fix all bows maybe becuase they all need a major overhaul.

**Comment by Dartan82** [comment]
Woah there asking for more than one zone to change that Lilithelle doesn't do every week is a tall order.

**Comment by Lilithelle** [comment]
We should replace arrow of slaying, with something like celestial purification on the crystallized tear of an astral deva.  Give it a power like celestial mercy, remove blind condition.

**Comment by jokkmeister** [comment]
> We should replace arrow of slaying, with something like celestial purification on the crystallized tear of an astral deva. Give it a power like celestial mercy, remove blind condition.

Do Rangers really need to be able to do everything? I mean the only little thing they don't have is probably cure blind. If you give them that too then they are complete. They are still too good being best offense, more than capable tanks and able to hide, it's too much as it is.
